scripts/poe/package.json

```diff
@@ -20,13 +20,15 @@
     "figlet": "1.2.0",
     "inquirer": "^6.2.1",
     "loglevel": "^1.4.1",
-    "request": "^2.88.0"
+    "messageformat": "^2.3.0",
+    "request": "^2.88.0",
+    "translation-checker": "file:../../../translation-checker"
```

Comment:
Don't forget to change this once the translation-checker is published!

Reply:
:man_facepalming: Oh yes in fact the intention was to commit "1.0.0" like in the medic-conf [PR](https://github.com/medic/medic-conf/blob/254-medic-conf-translation-check/package.json#L60), I'll change it.